styles.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+html {
+  font-family: sans-serif, 'Poppinsp';
+}
+
+body {
+  background-color: #e5e5e5;
+  height: 918px;
+  width: 375px;
```

Comment:
- [ ] One of the core requirements of this project is to make your design responsive, but giving an exact width to your body would make this impossible. kindly consider making your design responsive up to the desktop size which is `768px`.
  - Minimum mobile screen size should be `375`px. Errors in screen sizes less than `375`px should be ignored. Otherwise, mobile version should be responsive until the desktop screen size.